tiny_dnn/network.h

```diff
@@ -601,6 +601,7 @@ class network {
     const_iterator begin() const { return net_.begin(); }
     const_iterator end() const { return net_.end(); }
 
+#ifndef CNN_NO_SERIALIZATION
```

Comment:
Or is it better to nullify `network::save` and `network::load` method definitions when `#ifdef CNN_NO_SERIALIZATION`? So that errors can be detected at compile time.